pkg/heroku/heroku.go

```diff
@@ -269,3 +292,15 @@ func (lr *ListRange) SetHeader(req *http.Request) {
 	req.Header.Set("Range", hdrval)
 	return
 }
+
+type RequestHeaders struct {
+	CommitMessage string
+}
+
+func (r *RequestHeaders) Headers() http.Header {
+	headers := http.Header{}
+	if r.CommitMessage != "" {
+		headers.Set("Commit-Message", r.CommitMessage)
```

Comment:
this should be a constant, but wasn't sure if there was an existing best practice for where to put those